sdk/modelsrepository/Azure.IoT.ModelsRepository/src/RepositoryHandler.cs

```diff
@@ -76,10 +79,19 @@ public RepositoryHandler(Uri repositoryUri, ClientDiagnostics clientDiagnostics,
                     repositoryMetadata.Features != null &&
                     repositoryMetadata.Features.Expanded)
```

Comment:
so we currently don't persist the repositoryMetadata anywhere, just hold on to the expanded using a private field and update it on the next refresh if applicable ...

Reply:
Yes, because the thinking is currently the client only makes use of the expanded property. Once more metadata attributes are used in operations persisting the entire model would be more apt.